test/integration/join-test.js

```diff
@@ -107,15 +109,22 @@ testRingpopCluster({
 });
 
 testRingpopCluster({
-    size: 25
+    size: 25,
 }, 'mega cluster', function t(bootRes, cluster, assert) {
-    assert.equal(cluster.length, 25, 'cluser of 25');
-
-    cluster.forEach(function eachNode(node) {
-        assert.ok(node.isReady, 'node is bootstrapped');
+    cluster.forEach(function eachRingpop(ringpop) {
+            ringpop.gossip.stop();
     });
 
-    assert.end();
+    // wait 50 milliseconds to let pings die out, otherwise tchannel error 
```

Comment:
i dont think we should add any explicit waits in our test suite, if we dont have to. why isn't this satisfied by the "wait for convergence"?